Add eip-NFT_FT_with_royalty_standard.md

PowerStream3604 opened the pull request:
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

alita-moore commented:
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - File with name EIPS/eip-fractional_nft_with_royalty_standard.md is new and new files must be reviewed
	 - This PR requires review from one of [@MicahZoltu, @lightclient]
 @MicahZoltu @lightclient

wschwab commented on EIPS/eip-fractional_nft_with_royalty_standard.md:
```diff
@@ -0,0 +1,333 @@
+However, complying with the existing standards won't be an issue but additional gas might be needed for calling `functions`
+in this standard.
+
+## Reference Implementation
```
see general comment above

wschwab reviewed (commented):
I've left a few comments, but would also like to point out that the Solidity code included in the reference implementation does not compile, and does not appear to be complete. As a result, the intent of the EIP remains largely theoretical. For example, if the royalties are pushed to shareholders, who is pushing them, and wouldn't there be a large expenditure of gas?
I would recommend reviewing the code thoroughly.

PowerStream3604 commented:
> I've left a few comments, but would also like to point out that the Solidity code included in the reference implementation does not compile, and does not appear to be complete. As a result, the intent of the EIP remains largely theoretical. For example, if the royalties are pushed to shareholders, who is pushing them, and wouldn't there be a large expenditure of gas?
> I would recommend reviewing the code thoroughly.

Hi, thanks for the review. @wschwab 
I only revealed the key function for EIP 3602, I'll reveal the full code of it as soon as possible.
And to clarify, the ones pushing the royalty to shareholders are the people who need to pay the owners of the NFT.
As mentioned in the explanation above, for instance, the ones pushing are tenants, the ones withdrawing are owners of the building.

PowerStream3604 commented:
> There are numerous grammatical errors and incorrect formatting. Please revise the EIP and refer to EIP-1 for the correct structure.

I fixed the grammar errors and fixed headings reflecting EIP-1.
Please review this EIP and share your thoughts about the availability of this PR to be merged. @lightclient

lightclient reviewed (changes requested):
Hello @PowerStream3604, there are still too many errors to merge this. I recommend reviewing EIP-1 and other recently finalized EIPs to see the appropriate style + structure.

PowerStream3604 commented:
> Hello @PowerStream3604, there are still too many errors to merge this. I recommend reviewing EIP-1 and other recently finalized EIPs to see the appropriate style + structure.

I update the PR reflecting EIP1 & recently accepted EIPs. Please review this PR again.
@lightclient .
